pdns/dnsdist-cache.cc

```diff
@@ -201,7 +201,37 @@ void DNSDistPacketCache::insert(uint32_t key, const boost::optional<Netmask>& su
   }
 }
 
-bool DNSDistPacketCache::get(const DNSQuestion& dq, uint16_t consumed, uint16_t queryId, char* response, uint16_t* responseLen, uint32_t* keyOut, boost::optional<Netmask>& subnet, bool dnssecOK, uint32_t allowExpired, bool skipAging)
```

Comment:
Overload get function to return if key is valid.
I overloaded this function since its the same as the original get function but it just return true or false if the key is valid instead of relevant data.